code/modules/reagents/reagent_containers/spray.dm

```diff
@@ -34,7 +34,7 @@
 			to_chat(user, "<span class='notice'>[src] is full.</span>")
 			return
 
-		var/trans = A.reagents.trans_to(src, A:amount_per_transfer_from_this)
+		var/trans = A.reagents.trans_to(src, 50)

```

Comment:
this is a flat amount because having it be based on the amount transferred from the spray bottle would mean it would take forever to refill (since the amounts are 5 and 10).